scrnaseq.Rmd

```diff
@@ -881,9 +895,10 @@ if (param$norm == "RNA") {
   # Note: It is not guaranteed that all genes are successfully normalised with SCTransform. 
   #   Consequently, some genes might be missing from the SCT assay. 
   #   See: https://github.com/ChristophH/sctransform/issues/27
-  # Note: The performance of SCTransform can be improved by using 'glmGamPoi' instead of 'poisson' as method for initial parameter estimation.
+  # Note: The performance of SCTransform can be improved by using "glmGamPoi" instead of "poisson" as method for initial parameter estimation.
   sc = purrr::map(list_names(sc), function(n) { 
     SCTransform(sc[[n]], 
+                assay=param$assay_raw,
```

Comment:
Can you add this also to the `SCTransform` in the `integrate`chunk?

Reply:
Will do

Reply:
Done. Question is: I did not do it for `param$norm == "RNA"` ... Since for spatial data, we use `"SCT"`. -Theoretically- it would be possible though ... never tried it. The `param` section atm says: 

```
# Which normalisation should be used for analysis?
#   "RNA" or "SCT" for standard single-cell RNA-seq data
#   "SCT" for spatial data
param$norm = "RNA"
```

Reply:
Fixed for param$norm = "RNA"